packages/core/deployments/goerli/solcInputs/87af3bddb96fa5cfcfc99d0e6fd39498.json

```diff
@@ -0,0 +1,416 @@
+{
```

Comment:
Do we have to commit all of these generated files to the repo? It seems like all we should really need are the deployed addresses, like we store in networks/?

Reply:
My thought originally was to share the solc standard input JSON so that others could verify contracts, but whoever deploys contracts should ideally just use their own locally generated JSON. I'm going to remove everything in `deployments` (and add to `.gitignore`) but I'm going to add commentary about a script that we can use to easily export the local `deployments` data (incl. address and ABI) to the `networks/` directory.

For example `yarn hardhat export --export-all /networks/hardhat/deployed.json` makes a dictionary of contract addresses and ABI for each network that the user has local deployments data for.

[More on the export command](https://hardhat.org/plugins/hardhat-deploy.html#exporting-deployments)